ehrapy/preprocessing/_scanpy_pp_api.py

```diff
@@ -225,6 +232,23 @@ def neighbors(
         method: Use 'umap' [McInnes18]_ or 'gauss' (Gauss kernel following [Coifman05]_ with adaptive width [Haghverdi16]_) for computing connectivities.
                 Use 'rapids' for the RAPIDS implementation of UMAP (experimental, GPU only).
         metric: A known metric’s name or a callable that returns a distance.
+        transformer
```

Comment:
```suggestion
        transformer: Approximate kNN search implementation.
```
Or similar, as in this docstyle there can be a short description after the argument :)